layouts/partials/footer.html

```diff
@@ -9,7 +9,7 @@
 					</p>
 					<p>
 						{{ $lang := .Lang }}{{ $base := .Site.BaseURL }}{{ range .Site.Languages }}
-						<a class="nav-link" href="{{ $base }}{{ .Lang }}">{{ .LanguageName }}</a>
+						<a class="nav-link" href="{{ $base }}{{ .Lang }}" hreflang="{{ .Lang }}">{{ .LanguageName }}</a>
```

Comment:
AFIAU hreflang should only be used on `<meta>`  I think what you need is lang, as it is on default template.

```suggestion
						<a class="nav-link" href="{{ $base }}{{ .Lang }}" lang="{{ .Lang }}">{{ .LanguageName }}</a>
```

Reply:
`<a href="https://en.wikipedia.org/wiki/Ciao" lang="it">Ciao</a>` means the text inside the element is in Italian. `lang` specifies the language of the text content; `hreflang` specifies the language of the target document.

`hreflang` [can be used just fine on `a` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-hreflang).

Reply:
Does google understand the full language written out (I mean probably as they are google after all, but maybe some smaller search engines may not)  my understanding is that we should use the form en-US etc for hreflang.

Reply:
For either a `lang` or `hreflang` attribute, it should just be the ISO language code.

`.Lang` should give that code, not `.LanguageName`.

Reply:
Ok I guess it should have both then.

The language is the language name written in itself.